microsetta_private_api/db/patches/0139.sql

```diff
@@ -0,0 +1,28 @@
+-- May 13, 2024
+-- Create two tables for observation categories and observation errors list
+-- Add observations column to barcode_scans table
+CREATE TABLE observation_categories (
+    categories VARCHAR(255) PRIMARY KEY
+);
+
+INSERT INTO observation_categories (categories) VALUES ('Sample'), ('Swab'), ('Tube');
+
+CREATE TABLE observation_errors (
```

Comment:
```suggestion
CREATE TABLE sample_observations (
```

Some of these conditions are not errors in a strict sense, so it would be more appropriate to name the table "sample_observations."